app/src/main/res/values-ar/strings.xml

```diff
@@ -14,8 +14,8 @@
 <string name="action_about">حول</string>
 <string name="about_message_title">حول</string>
 <string name="about_message_text"><b>التسجيل السريع والآمن والموثوق - SQRL</b>\n\nالاصدارة: %s\n\nGithub مستودع:\n<a href="https://github.com/kalaspuffar/secure-quick-reliable-login">https://github.com/kalaspuffar/secure-quick-reliable-login</a>\n\nالمؤلفون:\nدانيال بيرسون (Daniel Persson)</string>
-<string name="decrypt_identity_message">You\'ve successfully scanned your secure identity. Please verify that the rescue data presented below corresponds to the data printed on your key page.</string>
-<string name="communication_incorrect_response">Incorrect response, the server didn\'t respond with an error code. Try again later.</string>
+<string name="decrypt_identity_message"> </string>
```

Comment:
Is this empty on purpose?

Reply:
Great catch. Thank you. Updated it and also updated the version numbers before release.